Sources/StreamChat/WebSocketClient/EventMiddlewares/TypingStartCleanupMiddleware_Tests.swift

```diff
@@ -38,7 +39,7 @@ class TypingStartCleanupMiddleware_Tests: XCTestCase {
     
     func test_stopTypingEvent_sentAfterTimeout() {
         // Simulate some user started typing
-        let otherUser = ChatUser(id: UUID().uuidString)
+        let otherUser = _ChatUser<NoExtraData>.mock(id: UUID().uuidString)
```

Comment:
```suggestion
        let otherUser: ChatUser = .mock(id: .unique)
```

nit: Should we provide a default value for `id` in ChatUser.mock if we are not interested in a specific value? 🤔

Reply:
✅